hawkbit-rest/hawkbit-ddi-api/src/main/java/org/eclipse/hawkbit/ddi/json/model/DdiActivateAutoConfirmation.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2022 Bosch.IO GmbH and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.hawkbit.ddi.json.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class DdiActivateAutoConfirmation {
+
+    @JsonProperty(required = false)
+    private final String initiator;
```

Comment:
I think we need to make this mandatory... for the reason, any user can activate auto confirmation and we will have no clue who exactly did it? For security concerns I think we need this info at least from TT prespective

Reply:
At the end we will persist and show the security user as well. The initiator field can be provided in case there us some external user triggered this operation.